src/main/java/com/rednavis/vaadin/showcase/backend/auth/jwt/JwtHeader.java

```diff
@@ -0,0 +1,12 @@
+package com.rednavis.vaadin.showcase.backend.auth.jwt;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/**
+ * Headers used for communicating JWT tokens
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class JwtHeader {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'JwtHeader' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/gh/rednavis/vaadin-showcase/pullRequest?prid=4872368)